### DISCUSSION TITLE
About Code of Conduct

### DISCUSSION THREAD

**Discussion by chykon** [intel/rohd#180] [General]
I don't know how important this is, but still:

1. Contributor Covenant has the [latest version 2.1](https://www.contributor-covenant.org/version/2/1/code_of_conduct/) and ROHD uses 1.4.
2. The contact email is `otc.community.care@intel.com`. Is that how it should be? For example, the [HAXM](https://github.com/intel/haxm) project has the [address of their team](https://github.com/intel/haxm/blob/master/CODE_OF_CONDUCT.md#enforcement).

**Comment by mkorbel1** [comment]
The Contributor Covenant for open source projects at Intel is based on central guidance.  It looks like guidance now allows for v2.1, so yes it would be good to update.  Thanks for checking!  I filed an issue here (https://github.com/intel/rohd/issues/182) and will update it soon.

The email address is also based on that same central guidance, but it looks like a new alias is being used with v2.1 so I'll update that as well.  If there are any code of conduct concerns, that email is a good place to contact and it will reach the appropriate project team promptly.  I'm not familiar with why HAXM is using a different email address for their code of conduct, perhaps they have some more specific needs for that project.

**Reply by comradekingu** [reply]
@mkorbel1 2.1 shifts power to "community leaders", away from "project maintainers". That isn't good.
A contact field doesn't codify behaviour, nor does it require pledging allegiance to it.
Notice the fundamental disconnect between not asking and projecting this agreement to be in place.

From memory, Intel sullied away 180 million dollars on stuff like this. It doesn't work.

**Reply by mkorbel1** [reply]
Thank you for your feedback, @comradekingu.  The code of conduct we have conveys the standards, values, expectations, and responsibilities which contributors should consider to guide their behavior with other community members.  Reports of violations will continue to be taken seriously.
